Cargo.lock

```diff
@@ -91,6 +91,11 @@ dependencies = [
  "memchr",
 ]
 
+[[package]]
+name = "amcl"
+version = "0.3.0"
+source = "git+https://github.com/darwinia-network/milagro_bls#421aa3af8a0618a46144560389debbe7e56609db"
```

Comment:
@hujw77 We need to cargo update to the latest commit?

Reply:
Yes, the commit is outdate.